api/openapi.yaml

```diff
@@ -149,6 +168,8 @@ components:
           description: >
             Whether the submission is still being graded,
             and the information may change soon.
+        test_progress:
+          $ref: "#/components/schemas/TestProgress"
```

Comment:
Could add description for when this field is present (only in the testing phase)